.circleci/config.yml

```diff
@@ -34,8 +44,16 @@ jobs:
       - store_artifacts:
           path: ~/repo/dist
           destination: dist
-      - deploy:
-          name: Deploy Git tag to npm registry
+      - persist_to_workspace:
+          root: ~/repo
+          paths: .
+
+  deploy:
+    <<: *defaults
+    steps:
+      - attach_workspace: *attach_options
+      - run:
+          name: Deploying package to npm registry
           command: |
             # CIRCLE_TAG indicates whether the job runs for the tag (as opposed to: the commit on the branch)
             if [ -z "${CIRCLE_TAG}" ]; then
```

Comment:
Can this be removed. In current workflow, deployment happens only in master with tags matching `/v.*/`.

Reply:
```

Reply:
The deployment already is configured for tagged builds.

      tags:
          only: /v.*/

 I think checking that again is not necessary.